tests/test_data/test_datasets/test_scannet_dataset.py

```diff
@@ -524,3 +525,7 @@ def test_seg_show():
     mmcv.check_file_exist(pts_file_path)
     mmcv.check_file_exist(gt_file_path)
     mmcv.check_file_exist(pred_file_path)
+    os.remove(pts_file_path)
```

Comment:
Suggest using `tempfile.TemporaryDirectory` in line 505 , thus we can simply use temp_dir.close() to release and destroy all files in the temp_dir?

Reply:
Great, I find example in another unittest file that destroys using os.remove. Will change to this fashion.